### PR TITLE
perf: remove transparent

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jung-han** [comment]
중요한건 아니지만.. 이후에 커밋 타입은 스쿼시 머지 할 때 performance에서 perf로 변경해서 부탁드립니다~

File: apps/chart/src/brushes/label.ts [hunk 1]
```diff
@@ -183,6 +183,8 @@ export function bubbleLabel(ctx: CanvasRenderingContext2D, model: BubbleLabelMod
   });
 
   if (text) {
+    ctx.shadowColor = 'rgba(0,0,0,0)';
```
Comment (jung-han) [inline apps/chart/src/brushes/label.ts:186]: 크게 중요한 건 아니지만 `rgba(0, 0, 0, 0)`으로 띄어쓰기가 작성되어있으면 좋을 것 같습니다

**Review by jung-han — APPROVED** [review]
완료입니다

**Comment by jungeun-cho** [comment]
> 중요한건 아니지만.. 이후에 커밋 타입은 스쿼시 머지 할 때 performance에서 perf로 변경해서 부탁드립니다~
넵, 변경했어요.

**Pull request merged**